### ISSUE TITLE
Unable to run with an empty nomad auth token

### ISSUE THREAD

**Issue by foozmeat** [robinovitch61/wander#52]
**Describe the bug**
The readme states that `-t ""` should work to send an empty auth token but I receive an error

**To Reproduce**
```shell
$ wander -a $NOMAD_ADDR -t ""
error: set WANDER_TOKEN env variable, wander_token in config file, or --token argument

$ wander               
Using config file: /Users/james/.wander.yaml
error: set WANDER_TOKEN env variable, wander_token in config file, or --token argument
```

**Expected behavior**
The `-t` should be optional or work with an empty string

**Desktop (please complete the following information):**
 - OS: macos
 - Version: 0.3.0

**Comment by robinovitch61** [comment]
Good catch! Thanks :)

**Comment by robinovitch61** [comment]
This should be fixed in v0.3.1 which is releasing now
